app/models/scheduled_message.rb

```diff
@@ -2,4 +2,18 @@ class ScheduledMessage < ActiveRecord::Base
   validates :body, presence: true
   validates :days_after_start, presence: true
   validates :title, presence: true
+
+  before_save :escape_specific_characters
+
+  def body
+    self[:body].html_safe if self[:body]
```

Comment:
are these `self`s necessary? 

also, it feels weird to me to re-define `body` since that is an attribute on this model. How about a method name like `html_safe_body`?

I tend to prefer this style when checking for presence:

```ruby
body && body.html_safe
```

but since we are validating the presence of body (above), shouldn't there always be one?